package.json

```diff
@@ -72,7 +72,6 @@
     "react-ellipsis-with-tooltip": "^1.0.7",
     "react-helmet": "^5.2.0",
     "react-redux": "^5.0.6",
-    "react-router": "^4.2.0",
     "react-router-bootstrap": "0.24.4",
     "react-router-dom": "^4.2.2",
```

Comment:
Shouldn't we remove this as well and use the react-router-dom provided by core?

Reply:
Yes I believe so

Reply:
Makes sense to me. I tested without and it's fine (did a fresh npm install)

Reply:
I believe it's still needed for dev-env, perhaps it should move to be a `devDependency`